### PR TITLE
pipe: Add missing Start and Process timestamps

### PR DESCRIPTION
**Pull request by Dridi**
And expose bereq.time in vcl_pipe now that we have some.

Refs #3562

### CODE REVIEW

**Review by nigoroll — APPROVED** [review]

**Comment by Dridi** [comment]
bugwash: Ok.

I will add a docfix to the VSL manual before merging.

**Comment by Dridi** [comment]
Rebased against master, with a note added to the VSL manual.

I plan to merge this once CI completes.

**Pull request merged**
